web_src/css/helpers.css

```diff
@@ -77,10 +77,14 @@ Gitea's private styles use `g-` prefix.
 .gt-overflow-x-auto { overflow-x: auto !important; }
 .gt-overflow-x-scroll { overflow-x: scroll !important; }
 .gt-overflow-y-hidden { overflow-y: hidden !important; }
+.gt-flex-wrap { flex-wrap: wrap !important; }
```

Comment:
IIRC there is already a `gt-fw`

Reply:
Yes, we should use it, and later migrate to tailwind-compatible classes.

Reply:
Interesting, I think I even looked for if a class already exists but didn't find it.
Must have overlooked it…